source/channels/make-calls.rst

```diff
@@ -20,6 +20,29 @@ You can share a call's link to use in a meeting request or share with other team
 
 The call link is valid for long as the channel is active. When a channel is archived or deleted the link will become invalid.
 
+Frequently asked questions
+--------------------------
+
+Is video supported?
+~~~~~~~~~~~~~~~~~~~
```

Comment:
Bringing some of these back here since I feel it's information useful to end-users. Anything more technical can be kept in the deployment page.